hooks/kubernetes_installer.py

```diff
@@ -22,12 +22,18 @@ def __init__(self, arch, version, kubernetes_file):
         self.version = version
         self.kubernetes_file = kubernetes_file
 
-    def install(self, output_dir='/opt/kubernetes/bin'):
+    def install(self, output_dir='/opt/kubernetes'):
         """ Install kubernetes binary files from the tar file or gsutil. """
```

Comment:
Yes the change away from gsutil is a bigger change.  The code still uses gsutil to get the binary files, and we use get to download the examples.  At this moment.